packages/fluentui/react-northstar/src/index.ts

```diff
@@ -1,9 +1,10 @@
-import * as themes from './themes';
-
 //
-// Theme
+// Themes
 //
-export { themes };
+export { default as teamsTheme } from './themes/teams';
```

Comment:
This would be a breaking change for downstream consumers importing `theme` currently. Make sure to add a CHANGELOG note.